R/roxygen2_templates.R

```diff
@@ -0,0 +1,30 @@
+# nocov start
```

Comment:
I'll continue working on this branch to correct the CI errors, feel free to start reviewing it.